### ISSUE TITLE
If paused, restart recording if incident flagged

### ISSUE THREAD

**Issue by brunosiqueira** [igarape/copcast-android#206]

**Comment by kamalg19** [comment]
@brunosiqueira Not sure if tested use-case is valid
steps:
1. Start mission
2. Pause mission for 10 mins
3. Long press volume up button. It flag incident and restart video recording. Works fine as expected part of this bug.
4. Again pause mission for 10 mins
5. Again long press volume up button. This doesn't flag incident and doesn't restart video recording.

Is behaviour during step 5 correct?

**Comment by brunosiqueira** [comment]
@kamalg19 we block a second incident flag within the recording video frame. After that video file is finished, we allow a second one. In average, we allow one incident every 5 minutes

**Comment by kamalg19** [comment]
So, what is expected behavior for step 5 in use case tested by me?

**Comment by brunosiqueira** [comment]
@kamalg19 you are right. We should restart the mission even if there was an incident registered in less than 5 minutes.
